### DISCUSSION TITLE
Search Page Overhaul

### DISCUSSION THREAD

**Discussion by shiftgeist** [jpochyla/psst#41] [General]
From #30:
> The search page needs an overhaul anyway. Ideas welcome!

Let's collect some ideas!

I'll start: How about...
- Tabs - If you click on a tab than the page scrolls down to the section.
- Filter - Lets you search for only one type (might make the search itself more complex).
- Overview - Every category (of 4) has a small section that show the top X results. You can click on some link to show all results of this type.

**Comment by jpochyla** [comment]
Thanks for starting this!

I'm not really a UI designer, but intuitively I would think that as we'll need to display the full results sooner or later anyway, I'd go for n. 3, maybe with a combination with n. 1.

Should we design the Artist page the same way? There is some overlap for sure.

**Reply by shiftgeist** [reply]
![search 2](https://user-images.githubusercontent.com/30648730/110707183-3b7fc080-81f9-11eb-87f6-5ca2fa6f189c.png)

some thoughts on nr 3

**Reply by jpochyla** [reply]
@shiftgeist Nice! Yes this is probably the way to go if we keep the wide layout. I was thinking about making the window thinner by default, with only one column in the main view (half of its size). I'm not sure about it yet., probably needs some prototyping, but with a thinner "explore" view we could put the currently playing context ("queue") to the right and have a proper two-pane layout. In such case we would probably add shortcut buttons to the top of the search page.

**Comment by pmeter** [comment]
Thank you for this amazing piece of software! If you'd have little time, there is one feature I'd like to be added to the search results. And thats a "Show more" button for each type of search results like 'Albums'. For me the functionality to access all albums of an artist is more important than how the results are presented GUI-wise. In the current version it seems not to be possible to see all albums of an artist.

**Reply by SO9010** [reply]
@jpochyla I this should be something we think about in #536
